src/_pytest/mark/structures.py

```diff
@@ -580,17 +580,21 @@ def __getattr__(self, name: str) -> MarkDecorator:
             # If the name is not in the set of known marks after updating,
             # then it really is time to issue a warning or an error.
             if name not in self._markers:
-                if self._config.option.strict_markers or self._config.option.strict:
-                    fail(
-                        f"{name!r} not found in `markers` configuration option",
-                        pytrace=False,
-                    )
-
                 # Raise a specific error for common misspellings of "parametrize".
                 if name in ["parameterize", "parametrise", "parameterise"]:
                     __tracebackhide__ = True
                     fail(f"Unknown '{name}' mark, did you mean 'parametrize'?")
 
+                if self._config.hasini("strict_markers"):
+                    strict_markers = self._config.getini("strict_markers")
+                else:
+                    strict_markers = self._config.getini("strict")
```

Comment:
While simple, this pattern repeats, how about a simple convenience method?


```suggestion
                strict_markers = self._config.getini_or("strict_markers", "strict")         
```

Not married to the name `getini_or`, other suggestions:

* `getini_fallback`
* `getini_or_else`

Reply:
I'll comment about this in the issue.